Failed to Activate Extension: Issue migrating runtime folder dest already exists

## Description
Seeing an error when opening the extension. Had it working a few weeks ago, but recently it stopped. Tried removing VS Code and it's support directories, then re-installing it, as well as upgrading npm to 6.x (from 5.6), but same error. Also tried going back to VS Code v1.32 (was on 1.38).

## Expected Behavior
Expect the extension to start up properly

## Actual Behavior
See Log

## Steps to Reproduce
<!--- Provide a link to a live example, or an unambiguous set of steps to -->
<!--- Reproduce this bug include code to reproduce, if relevant -->
1. Install extension
2. Restart VS Code
3. Click extension icon to open and receive error: `Failed to activate extension: Issue migrating runtime folder dest already exists.`

## Log File
```
[9/7/2019 4:18:22 PM] [INFO] Starting IBM Blockchain Platform Extension
[9/7/2019 4:18:22 PM] [INFO] Attempting to require dependency: grpc
[9/7/2019 4:18:22 PM] [INFO] Error requiring dependency: The gRPC binary module was not installed. This may be fixed by running "npm rebuild"
Original error: Cannot find module '/Users/dkneisly/.vscode/extensions/ibmblockchain.ibm-blockchain-platform-1.0.9/node_modules/grpc/src/node/extension_binary/node-v64-darwin-x64-unknown/grpc_node.node'
[9/7/2019 4:18:24 PM] [INFO] Updating native node modules
[9/7/2019 4:18:24 PM] [INFO] Rebuilding native node modules
[9/7/2019 4:18:26 PM] [INFO] > grpc@1.22.2 install /Users/dkneisly/.vscode/extensions/ibmblockchain.ibm-blockchain-platform-1.0.9/node_modules/grpc
[9/7/2019 4:18:26 PM] [INFO] > node-pre-gyp install --fallback-to-build --library=static_library
[9/7/2019 4:18:27 PM] [INFO] node-pre-gyp
[9/7/2019 4:18:27 PM] [INFO]  
[9/7/2019 4:18:27 PM] [INFO] WARN Using request for node-pre-gyp https download 
[9/7/2019 4:18:27 PM] [INFO] [grpc] Success: "/Users/dkneisly/.vscode/extensions/ibmblockchain.ibm-blockchain-platform-1.0.9/node_modules/grpc/src/node/extension_binary/electron-v3.1-darwin-x64-unknown/grpc_node.node" is installed via remote
[9/7/2019 4:18:28 PM] [INFO] grpc@1.22.2 /Users/dkneisly/.vscode/extensions/ibmblockchain.ibm-blockchain-platform-1.0.9/node_modules/grpc
[9/7/2019 4:18:28 PM] [INFO] Updating grpc
[9/7/2019 4:18:28 PM] [SUCCESS] Finished updating native node modules
[9/7/2019 4:18:28 PM] [INFO] Rewriting activation events
[9/7/2019 4:18:28 PM] [INFO] Clearing extension cache
[9/7/2019 4:18:28 PM] [INFO] Restoring command registry
[9/7/2019 4:18:28 PM] [INFO] Migrating local runtime manager
[9/7/2019 4:18:28 PM] [WARNING] Attempting to teardown old local Fabric from version <= 0.3.3
[9/7/2019 4:18:28 PM] [WARNING] Any error messages from this process can be safely ignored (for example, container does not exist
[9/7/2019 4:18:28 PM] [INFO] # Remove the Docker containers for the runtime components.
[9/7/2019 4:18:28 PM] [INFO] docker rm -f fabricvscodelocalfabric_peer0.org1.example.com
[9/7/2019 4:18:28 PM] [INFO] Error: No such container: fabricvscodelocalfabric_peer0.org1.example.com
[9/7/2019 4:18:28 PM] [INFO] docker rm -f fabricvscodelocalfabric_ca.example.com
[9/7/2019 4:18:28 PM] [INFO] Error: No such container: fabricvscodelocalfabric_ca.example.com
[9/7/2019 4:18:28 PM] [INFO] docker rm -f fabricvscodelocalfabric_orderer.example.com
[9/7/2019 4:18:28 PM] [INFO] Error: No such container: fabricvscodelocalfabric_orderer.example.com
[9/7/2019 4:18:28 PM] [INFO] docker rm -f fabricvscodelocalfabric_couchdb
[9/7/2019 4:18:28 PM] [INFO] Error: No such container: fabricvscodelocalfabric_couchdb
[9/7/2019 4:18:28 PM] [INFO] docker rm -f fabricvscodelocalfabric_logs
[9/7/2019 4:18:28 PM] [INFO] Error: No such container: fabricvscodelocalfabric_logs
[9/7/2019 4:18:28 PM] [INFO] # Remove the Docker containers and images for any chaincode images.
[9/7/2019 4:18:28 PM] [INFO] docker ps -aq --filter "name=fabricvscodelocalfabric-*" | xargs docker rm -f
[9/7/2019 4:18:28 PM] [INFO] docker images -aq "fabricvscodelocalfabric-*" | xargs docker rmi -f
[9/7/2019 4:18:28 PM] [INFO] # Remove the Docker volumes.
[9/7/2019 4:18:28 PM] [INFO] docker volume rm -f fabricvscodelocalfabric_peer0.org1.example.com
[9/7/2019 4:18:28 PM] [INFO] fabricvscodelocalfabric_peer0.org1.example.com
[9/7/2019 4:18:28 PM] [INFO] docker volume rm -f fabricvscodelocalfabric_ca.example.com
[9/7/2019 4:18:28 PM] [INFO] fabricvscodelocalfabric_ca.example.com
[9/7/2019 4:18:28 PM] [INFO] docker volume rm -f fabricvscodelocalfabric_orderer.example.com
[9/7/2019 4:18:28 PM] [INFO] fabricvscodelocalfabric_orderer.example.com
[9/7/2019 4:18:28 PM] [INFO] docker volume rm -f fabricvscodelocalfabric_couchdb
[9/7/2019 4:18:29 PM] [INFO] fabricvscodelocalfabric_couchdb
[9/7/2019 4:18:29 PM] [INFO] docker volume rm -f fabricvscodelocalfabric_logs
[9/7/2019 4:18:29 PM] [INFO] fabricvscodelocalfabric_logs
[9/7/2019 4:18:29 PM] [INFO] # Remove the Docker network.
[9/7/2019 4:18:29 PM] [INFO] docker network rm fabricvscodelocalfabric_basic
[9/7/2019 4:18:29 PM] [INFO] Error: No such network: fabricvscodelocalfabric_basic
[9/7/2019 4:18:29 PM] [INFO] exit 0
[9/7/2019 4:18:29 PM] [WARNING] Finished attempting to teardown old local Fabric from version <= 0.3.3
[9/7/2019 4:18:29 PM] [ERROR] Failed to activate extension: Error: Issue migrating runtime folder dest already exists
```

## Your Environment
<!--- Include as many relevant details about the environment you experienced the bug in -->
* IBM Blockchain Platform Extension Version: 1.0.9
* Visual Studio Code version: 1.32 or 1.38
* Environment name and version (e.g. Node.js v8.12.0, npm v6.4.1): Node.js v8.9.4, npm v6.11.3
* Operating System and version: macOS 10.14.6

--- a/src/fabric/FabricRuntimeManager.ts
+++ b/src/fabric/FabricRuntimeManager.ts
@@ -208,7 +208,10 @@ export class FabricRuntimeManager {
         if (exists) {
             try {
                 const newPath: string = path.join(extDir, 'environments', FabricRuntimeUtil.LOCAL_FABRIC);
-                await fs.move(runtimesExtDir, newPath);
+                const newPathExists: boolean = await fs.pathExists(newPath);
+                if (!newPathExists) {
+                    await fs.move(runtimesExtDir, newPath);
+                }
             } catch (error) {
                 throw new Error(`Issue migrating runtime folder ${error.message}`);
             }

--- a/test/fabric/FabricRuntimeManager.test.ts
+++ b/test/fabric/FabricRuntimeManager.test.ts
@@ -491,7 +491,7 @@ describe('FabricRuntimeManager', () => {
             let extDir: string = vscode.workspace.getConfiguration().get(SettingConfigurations.EXTENSION_DIRECTORY);
             extDir = UserInputUtil.getDirPath(extDir);
 
-            sandbox.stub(fs, 'pathExists').resolves(true);
+            sandbox.stub(fs, 'pathExists').onFirstCall().resolves(true).onSecondCall().resolves(false);
             const moveStub: sinon.SinonStub = sandbox.stub(fs, 'move').resolves();
 
             await runtimeManager.migrate(version);
@@ -509,12 +509,23 @@ describe('FabricRuntimeManager', () => {
             moveStub.should.not.have.been.called;
         });
 
+        it ('should not move if environment folder already exists', async () => {
+            let extDir: string = vscode.workspace.getConfiguration().get(SettingConfigurations.EXTENSION_DIRECTORY);
+            extDir = UserInputUtil.getDirPath(extDir);
+
+            sandbox.stub(fs, 'pathExists').resolves(true);
+            const moveStub: sinon.SinonStub = sandbox.stub(fs, 'move').resolves();
+
+            await runtimeManager.migrate(version);
+            moveStub.should.not.have.been.called;
+        });
+
         it('should handle error moving', async () => {
             const error: Error = new Error('some error');
             let extDir: string = vscode.workspace.getConfiguration().get(SettingConfigurations.EXTENSION_DIRECTORY);
             extDir = UserInputUtil.getDirPath(extDir);
 
-            sandbox.stub(fs, 'pathExists').resolves(true);
+            sandbox.stub(fs, 'pathExists').onFirstCall().resolves(true).onSecondCall().resolves(false);
             const moveStub: sinon.SinonStub = sandbox.stub(fs, 'move').throws(error);
 
             await runtimeManager.migrate(version).should.eventually.be.rejectedWith(`Issue migrating runtime folder ${error.message}`);

